ui/imports/Emoji.qml

```diff
@@ -10,23 +10,25 @@ QtObject {
         Twemoji.twemoji.base = base
         Twemoji.twemoji.ext = ".png"
         Twemoji.twemoji.size = size
-        return Twemoji.twemoji.parse(text)
+        return Twemoji.twemoji.parse(text, {
+            attributes: function() { return { width: "20", height: "20" }}
```

Comment:
No, because size specifies the source image to retrieve from the filesystem, but the width and height are attributes on the `<img/>`, which essentially downscales the emoji but triples its pixel density (making it sharper on high density screens).

Reply:
What happens with the real big emojis then? When you put an emoji alone as a message, it should show the 72x72 image

Reply:
Good point, added a `renderSize` parameter.